From: Steve Langasek <vorlon@debian.org>
Date: Mon, 11 Sep 2023 14:00:42 -0600
Subject: define PATH_MAX for compatibility when it's not already set

Bug-Debian: http://bugs.debian.org/552043

Some platforms, such as the Hurd, don't set PATH_MAX.  Set a reasonable
default value in this case.
---
 tests/tst-dlopen.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/tst-dlopen.c b/tests/tst-dlopen.c
index 7092716..535ee1c 100644
--- a/tests/tst-dlopen.c
+++ b/tests/tst-dlopen.c
@@ -16,6 +16,11 @@
 #include <limits.h>
 #include <sys/stat.h>
 
+/* Hurd compatibility */
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 /* Simple program to see if dlopen() would succeed. */
 int main(int argc, char **argv)
 {
